modules/ROOT/pages/rest-sdk/edits/features.adoc

```diff
@@ -0,0 +1,69 @@
+= Overview
```

Comment:
Apologies: such a short title that I edited out of order!
Again, "Overview" is a super generic title: overview of what? all of REST SDK? something specific about REST SDK? Do we mean How REST SDK Works?
I think what you want her is what the .adoc title implies: features. And features that are different from REST Connect.
So...maybe something like...:
```suggestion
= REST SDK Features
```
or
```suggestion
= REST SDK Metadata-Driven Features
```
or
```suggestion
= REST SDK Feature Improvements Over Rest Connect
```
or....  Whatever your main focus is.